pkg/util/util.go

```diff
@@ -24,3 +27,18 @@ func GetNamespace() (string, error) {
 	}
 	return namespaceKubevirt, nil
 }
+
+func IsNvidiaGpuVmi(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
Is it possible to avoid using `Nvidia` explicitly? This code should be generic.

Reply:
Removed